openedx_events/enterprise/data.py

```diff
@@ -94,3 +94,71 @@ class LedgerTransaction(BaseLedgerTransaction):
     parent_content_key = attr.ib(type=str, default=None)
     fulfillment_identifier = attr.ib(type=str, default=None)
     reversal = attr.ib(type=LedgerTransactionReversal, default=None)
+
+
+@attr.s(frozen=True)
+class EnterpriseCustomerUser:
+    """
+    Defines the common attributes of enterprise fulfillment classes.
+    """
+
+    id = attr.ib(type=int)
+    created = attr.ib(type=datetime)
+    modified = attr.ib(type=datetime)
+    enterprise_customer_uuid = attr.ib(type=UUID)
+    user_id = attr.ib(type=int)
+    active = attr.ib(type=bool)
+    linked = attr.ib(type=bool)
+    is_relinkable = attr.ib(type=bool)
+    invite_key = attr.ib(type=UUID)
+    should_inactivate_other_customers = attr.ib(type=bool)
+
+
+@attr.s(frozen=True)
+class EnterpriseCourseEnrollment:
+    """
+    Defines the common attributes of enterprise fulfillment classes.
+    """
+
+    id = attr.ib(type=int)
+    created = attr.ib(type=datetime)
+    modified = attr.ib(type=datetime)
+    enterprise_customer_user = attr.ib(type=EnterpriseCustomerUser)
+    course_id = attr.ib(type=CourseKey)
```

Comment:
[sanity check] for my own edification, does `CourseKey` adequately handle course _run_ keys?

Reply:
Yes, this is an age old question, but my understanding is that CourseKey from opaque-keys is supposed to represent a course run, and AFAIK it can't even accurately represent a course key sans run.